frontend/src/app/cycle-admin/cycle-creation-form/cycle-creation-form.component.html

```diff
@@ -7,14 +7,14 @@
     </mat-form-field>
 
     <mat-form-field class="halfWidth leftDate">
-      <input [matDatepicker]="startDatePicker" formControlName="startDate" matInput placeholder="{{ 'cycle-creation-form.start-date' | translate }}">
+      <input [matDatepicker]="startDatePicker" (dateChange)="dateChangeHandler($event)" formControlName="startDate" matInput placeholder="{{ 'cycle-creation-form.start-date' | translate }}">
       <mat-datepicker-toggle [for]="startDatePicker" matSuffix></mat-datepicker-toggle>
       <mat-datepicker #startDatePicker></mat-datepicker>
       <mat-error><app-form-error [control]="cycleForm.get('startDate')"></app-form-error></mat-error>
 
     </mat-form-field>
     <mat-form-field class="halfWidth rightDate">
-      <input [matDatepicker]="endDatePicker" formControlName="endDate" matInput placeholder="{{ 'cycle-creation-form.end-date' | translate }}">
+      <input matInput [matDatepicker]="$any(endDatePicker)" [min]="firstAvailableDate" formControlName="endDate" placeholder="{{ 'cycle-creation-form.end-date' | translate }}">
```

Comment:
Wofür wird das $any() benötigt?
```suggestion
      <input matInput [matDatepicker]="endDatePicker" [min]="firstAvailableDate" formControlName="endDate" placeholder="{{ 'cycle-creation-form.end-date' | translate }}">
```